### PR TITLE
Proposal of OpenKruise project for LFX 2024 Jun-Aug

### PR DESCRIPTION
**Pull request by chrisliu1995**

### CODE REVIEW

File: programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md [hunk 1]
```diff
@@ -297,3 +297,13 @@ Onboarding new end users into a sophisticated system like Knative Eventing prese
...
 
+### OpenKruise 
+
+#### Enhancement for Kruise-Game Dashboard
+
+- Description: The OpenKruiseGame Dashboard is presently in its basic form, and we aim to significantly expand its functionality going forward. We plan to introduce features such as the ability to filter game servers and perform batch updates on them.
+- Expected Outcome: new release Dashboard which supports searching, querying, updating objects in batch.
+- Recommended Skills: Kubernetes, nodejs, javascript
+- Mentor(s):
+  - Qiuyang Liu (@chrisliu1995, chrisliu1995@163.com)
```
Comment (aliok) [inline programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md:308]: @chrisliu1995 can you find a co-mentor? It makes everyones lives easier. We strongly encourage mentors to find a co-mentor.

File: programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md [hunk 2]
```diff
@@ -297,3 +297,13 @@ Onboarding new end users into a sophisticated system like Knative Eventing prese
...
+- Upstream Issue: https://github.com/openkruise/kruise-game/issues/139
```
Comment (aliok) [inline programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md:309]: I've checked the upstream issue. It seems blank. Can you add some details there too?

**Review by aliok — COMMENTED** [review]
Thanks for the proposal @chrisliu1995.

I've left 2 comments.

Also, can you ask for a 👍 or an approval from one of the maintainers of OpenKruise listed here? https://raw.githubusercontent.com/cncf/foundation/main/project-maintainers.csv

File: programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md (hunk 2 above)
Comment (chrisliu1995) [reply programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md:310]: Of course. Added now

File: programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md (hunk 1 above)
Comment (chrisliu1995) [reply programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md:308]: Find a co-mentor @ringtail

**Comment by chrisliu1995** [comment]
> Thanks for the proposal @chrisliu1995.
> 
> I've left 2 comments.
> 
> Also, can you ask for a 👍 or an approval from one of the maintainers of OpenKruise listed here? https://raw.githubusercontent.com/cncf/foundation/main/project-maintainers.csv

@furykerry

**Comment by aliok** [comment]
As the final step, can @ringtail approve the PR or give a 👍 please?
Then we can merge.

**Comment by ringtail** [comment]
> As the final step, can @ringtail approve the PR or give a 👍 please? Then we can merge.

Thanks @aliok

**Review by aliok — APPROVED** [review]

**Pull request merged**